snuba/datasets/outcomes.py

```diff
@@ -149,4 +151,17 @@ def __init__(self):
         super().__init__(
             dataset_schemas=dataset_schemas,
             table_writer=table_writer,
+            time_group_columns={
+                'time': 'timestamp',
+            },
+            time_parse_columns=('timestamp',)
         )
+
```

Comment:
Are you planning to support these:
- get_split_query_spec
- get_prewhere_keys

Reply:
`get_split_query_spec`, not for now. Good point about `get_prewhere_keys` - I'll add those.